includes/upgrade.php

```diff
@@ -47,10 +61,77 @@ private static function get_database_schema_sql(): string {
 				`event_id` int(10) NOT NULL COMMENT 'Post_ID of the translation_event post in the wp_posts table',
 				`user_id` int(10) NOT NULL COMMENT 'ID of the user who is attending the event',
 				`is_host` tinyint(1) default 0 not null comment 'Whether the user is a host of the event',
+				`is_new_contributor` tinyint(1) default 0 not null comment 'Whether the user is a new translation contributor',
 			PRIMARY KEY (`translate_event_attendees_id`),
 			UNIQUE KEY `event_per_user` (`event_id`,`user_id`),
 			INDEX `user` (`user_id`)
 			) COMMENT='Attendees of events';
 		";
 	}
+
+	/**
+	 * Set is_new_contributor in attendees table for all events.
+	 */
+	private static function v3_set_is_new_contributor(): void {
+		global $wpdb, $gp_table_prefix;
+
+		$query = new WP_Query(
+			array(
+				'post_type'   => Translation_Events::CPT,
+				'post_status' => 'publish',
+			)
+		);
+
+		$events              = $query->get_posts();
+		$event_repository    = Translation_Events::get_event_repository();
+		$attendee_repository = Translation_Events::get_attendee_repository();
+
+		foreach ( $events as $post ) {
+			$event = $event_repository->get_event( $post->ID );
+			if ( ! $event ) {
+				continue;
+			}
+
+			$attendees = $attendee_repository->get_attendees( $event->id() );
+
+			foreach ( $attendees as $attendee ) {
+				if ( ! $attendee->is_contributor() ) {
```

Comment:
I think it should be possible that someone did not end up contributing but was a first contributor at the time

Reply:
Good point. I didn't consider that we show is_new_contributor in the list of non-contributing attendees. Fixed.